src/pages/home/HeaderView.js

```diff
@@ -271,21 +271,23 @@ HeaderView.propTypes = propTypes;
 HeaderView.displayName = 'HeaderView';
 HeaderView.defaultProps = defaultProps;
 
-export default compose(
-    withWindowDimensions,
-    withLocalize,
-    withOnyx({
-        guideCalendarLink: {
-            key: ONYXKEYS.ACCOUNT,
-            selector: (account) => (account && account.guideCalendarLink) || null,
-            initialValue: null,
-        },
-        parentReport: {
-            key: ({report}) => `${ONYXKEYS.COLLECTION.REPORT}${report.parentReportID || report.reportID}`,
-            selector: reportWithoutHasDraftSelector,
-        },
-        session: {
-            key: ONYXKEYS.SESSION,
-        },
-    }),
-)(HeaderView);
+export default memo(
```

Comment:
Memo'ing this prevent updating the name and image of the workspace inside header [[more context](https://github.com/Expensify/App/issues/30724)]